src/js/select2/i18n/uk.js

```diff
@@ -0,0 +1,40 @@
+define(function () {
+  function ending(count, one, couple, more) {
+    if ([11,12,13,14].indexOf(count % 100) !== -1) {
+      return more;
+    }
+    if (count % 10 === 1) {
+      return one;
+    }
+    if ([2,3,4].indexOf(count % 10) !== -1) {
+      return couple;
+    }
+    return more;
+  }
+
+  return {
+    errorLoading: function () {
+      return 'Неможливо завантажити результати';
+    },
+    inputTooLong: function (args) {
+      var overChars = args.input.length - args.maximum;
+      return 'Будь ласка, видаліть ' + overChars + ending(args.maximum, 'літеру', 'літери', 'літер');
```

Comment:
Should there be a space between `overChars` and `ending`? Otherwise you have the number directly next to the word.